tsconfig.json

```diff
@@ -10,8 +10,13 @@
     "noEmitOnError": true,
     "skipDefaultLibCheck": true,
     "skipLibCheck": true,
-    "outDir": "./dist",
+    "outDir": "../hardhat-tutorial/node_modules/hardhat-gobind",
```

Comment:
?

Reply:
this is for local testing purposes, I'll change it before publishing to NPM or pulling the branch